py-server/app.yaml

```diff
@@ -1,6 +1,11 @@
 runtime: python312
 instance_class: F4_1G
 
+automatic_scaling:
+  target_cpu_utilization: 0.65
```

Comment:
could you clarify why we need this? IIUC,  there will always be exactly one instance regardless of TPU utilization?